src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/transformer/ReferralTransformers.kt

```diff
@@ -40,7 +40,7 @@ fun ReferralEntity.toApi(): ApiReferral = ApiReferral(
 )
 
 fun StaffEntity.toApi() = StaffDetail(
-  staffId = staffId!!,
+  staffId = staffId?.or("0".toBigInteger())!!,
```

Comment:
Is setting a default value of '0' a good idea? Can we not have null as the default?

Reply:
Good point, updated it to be optional. So it will take null values now